canonical-web-and-design/maas-ui
# feat(ui): tag details modal

## Done

- Display the tag details when click on a chip in the tag form.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list and select some machines with tags.
- Open tag form.
- Click on a tag and you should see the tag details in a modal.

## Fixes

Fixes: canonical-web-and-design/app-tribe#686.

## Screenshot

![Uploading Screen Shot 2022-03-30 at 12.55.42 pm.png…]()
